Server/Content/Sources/App/routes.swift

```diff
@@ -0,0 +1,24 @@
+import Fluent
+import Vapor
+
+func routes(_ app: Application) throws {
+    app.get { req in
+        return "It works!"
+    }
+
+    app.get("hello") { req -> String in
+        return "Hello, world!"
+    }
+
+    let textController = TextController()
+    app.get("texts", use: textController.all)
+    app.get("texts", ":textID", use: textController.getById)
```

Comment:
You should use one style for naming acronyms (e.g. API, ID & etc.), you either need to capitalize them all the time, or use camelCase.
Here in just one line you have used them differently: text**ID** and getBy**Id** - your naming should be consistent, because other people are going to read your code, if you care about them - choose names wisely & be consistent :)

P.S. Here I suggest that we use camelCase all the time for acronyms